rest/servers/spring/src/main/java/com/connexta/store/rest/spring/StoreApi.java

```diff
@@ -175,4 +179,68 @@
           QuarantineRequest quarantineRequest) {
     return new ResponseEntity<>(HttpStatus.NOT_IMPLEMENTED);
   }
+
+  @ApiOperation(
```

Comment:
✏️ I suggest moving this method to the top of the class to better align with the order of the workflow.